packages/wdio-sync/webdriverio-core.d.ts

```diff
@@ -1333,6 +1333,18 @@ declare namespace WebdriverIO {
         ): boolean;
     }
 
+    type MultiRemoteBrowserReference = Record<string, BrowserObject>
+    interface MultiRemoteBrowser extends Browser {
+        /**
+         * multiremote browser instance names
+         */
+        instances: string[];
+        /**
+         * flag to indicate multiremote browser session
+         */
+        isMultiremote: true;
+    }
+    type MultiRemoteBrowserObject = MultiRemoteBrowser & MultiRemoteBrowserReference
```

Comment:
Union type is the only way I've found to define typings for multiremote browser access such as `global.browser[browserName]`. See https://stackoverflow.com/a/57932353/3439960 for similar issue and https://www.typescriptlang.org/docs/handbook/interfaces.html for why extending Dictionary types does not work